lib/patterns.rb

```diff
@@ -58,7 +55,6 @@ module Patterns
     resolv
     resolv-replace
     rinda
-    ruby
```

Comment:
Only this is reserved as of now. Do need ownership of this?

Reply:
Ah, Sorry. This pr is obsoleted. 

https://github.com/rubygems/rubygems.org/pull/2304

FYI: I got the ownership of `ruby` namaspace from the current name-holder.

Reply:
Nice. please send new PR if you would like us to remove it from the reserved namespace.